samples/nrf9160/nrf_cloud_rest_cell_pos/README.rst

```diff
@@ -0,0 +1,98 @@
+.. _nrf_cloud_rest_cell_pos_sample:
+
+nRF9160: nRF Cloud REST Cellular Positioning
+############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The REST Cellular Positioning sample demonstrates how to use the `nRF Cloud REST API`_ to use nRF Cloud's cellular positioning location service on your device.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf9160dk_nrf9160_ns
+
+The sample requires an `nRF Cloud`_ account.
+The sample requires one of the following:
+* The device is provisioned and associated with an `nRF Cloud`_ account. The sample supports JITP provisioning via REST.
+* A private key is installed on the device and its associated public key is registered with an `nRF Cloud`_ account.
+
+.. note::
+   This sample requires modem firmware v1.3.x or later.
+
+Overview
+********
+
+After the sample initializes and connects to the network, it will enter single-cell mode and send a single-cell location request to nRF Cloud using network data obtained from the ref:`Modem Info <modem_info_readme>` library.
+To enable multi-cell mode, press button 1.
+In multi-cell mode, the sample will request a neighbor cell measurement using the ref:`LTE Link Control <lte_lc_readme>` library.
+If neighbor cell data is measured, the sample will send a multi-cell location request to nRF Cloud. Otherwise, the request will be single-cell.
+In either mode, the sample will send a new location request if a change in cell ID is detected.
```

Comment:
```suggestion
After the sample initializes and connects to the network, it enters single-cell mode and sends a single-cell location request to nRF Cloud using network data obtained from the ref:`modem_info_readme` library.
To enable multi-cell mode, press **button 1**.
In multi-cell mode, the sample requests for neighbor cell measurement using the ref:`lte_lc_readme` library.
If neighbor cell data is measured, the sample sends a multi-cell location request to nRF Cloud.
Otherwise, the request is single-cell.
In either mode, the sample sends a new location request if a change in cell ID is detected.
```